sys/arm64/arm64/machdep.c

```diff
@@ -454,16 +470,21 @@ exec_setregs(struct thread *td, struct image_params *imgp, uintcap_t stack)
 	tf->tf_elr = imgp->entry_addr;
 }
 
+#ifndef MORELLO
 /* Sanity check these are the same size, they will be memcpy'd to and fro */
```

Comment:
Uh, are these assertions not rather important as them failing means there aren't capabilities in the types?

Reply:
They will be fixed by a later change. They are to check the memcpy below them is valid. As there is no memcpy in the Morello case they can be ignored.